File: travelWorldAPP/app/models/api-models/Country.ts
/// <reference path="../../configs/_all.ts" />
module Travel_World {
    export class Country {
        public id: number;
        public name: string;
        public continentId: number;
        public continent: Continent;

        public cities: Array<City>;

        constructor() {
        }
    }
} 

File: TravelWorldAPI/ModernWebStore.Domain/Entities/Country.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace ModernWebStore.Domain.Entities
{
    public class Country
    {
        public int Id { get; set; }
        public string Name { get; set; }

        public int ContinentId { get; set; }
        public Continent Continent { get; set; }

        public ICollection<City> Cities { get; set; }
    }
}
